backend/.pylintrc

```diff
@@ -1,2 +1,3 @@
 [MASTER]
 extension-pkg-whitelist=pydantic
+ignore-paths=btrixcloud/migrations/migration_0028_page_files_errors.py
```

Comment:
Added because pylint started throwing an invalid-name error that could not be ignored on this file :/